glutton.go

```diff
@@ -256,7 +256,14 @@ func (g *Glutton) Shutdown() (err error) {
 // OnErrorClose prints the error, closes the connection and exits
 func (g *Glutton) onErrorClose(err error, conn net.Conn) {
 	if err != nil {
+		host, port, err := net.SplitHostPort(conn.RemoteAddr().String())
+		if err != nil {
+			g.logger.Error(fmt.Sprintf("[glutton ] error: %v", err))
+		}
 		g.logger.Error(fmt.Sprintf("[glutton ] error: %v", err))
+		g.logger.Debug(
+			fmt.Sprintf("Closing connection on host %s and port %s", host, port),
```

Comment:
But if we reach the timeout we are not returning an error. So we would never see this log message if the connection has reached its timeout.

Reply:
This might work for the default [handler](https://github.com/mushorg/glutton/blob/e5e4c564daab44bc6ccb9e25acb2b499b7244845/protocols.go#L66), what about the others?

Reply:
Yeah working on those as well :)

Reply:
I check this with a connection timeout but it did not print the debug statement there! Is this the correct place where the connection is actually closed?